### DISCUSSION TITLE
Swiper with parallax isn't working in demo and in fresh project.

### DISCUSSION THREAD

**Discussion by DeepKumbhare85** [nolimits4web/swiper#6906] [Q&A]
Hello, 

I am working on a project where I require swiper with parallax.I am using swiper element.I also checked out swiper element demo for parallax. In demo image is not sliding. parallax is working properly in vue & react demo.but is not working in element demo.Here is a link for swiper element demo: https://codesandbox.io/p/sandbox/swiper-parallax-forked-j6jpnf?file=%2Findex.html%3A11%2C7

Is there any mistake with implementation or this is a bug ??

**Comment by nolimits4web** [comment]
I think it is a bug, will be fixed soon with next update

**Reply by DeepKumbhare85** [reply]
Thank you for the response.I have created a issue regarding this: https://github.com/nolimits4web/swiper/issues/6910
